doc/parameters.md

```diff
@@ -4,124 +4,134 @@ APIcast v2 has a number of parameters configured as environment variables that c
 
 Note that when deploying APIcast v2 with OpenShift, some of thee parameters can be configured via OpenShift template parameters. The latter can be consulted directly in the [template](https://raw.githubusercontent.com/3scale/apicast/master/openshift/apicast-template.yml).
 
-- `APICAST_CUSTOM_CONFIG`
+#### `APICAST_CUSTOM_CONFIG`
```

Comment:
I'd go for:

# Describing Document (APIcast parameters)

## Section (Environment variables)

## Item (APICAST_CUSTOM_CONFIG)

Which should leave enough space for documenting some other parameters and configuration in the future.